src/lib/style/components/_rss-feed-cards.scss

```diff
@@ -35,12 +35,26 @@
 }
 .RssFeed {
   padding: 15px 0px;
+
+  @media (min-width: 801px) {
   .RssFeedItems {
     display: flex;
     flex-wrap: wrap;
     justify-content: center;
     .RssFeedItem {
-      max-width: 340px;
+      width: 30%;
```

Comment:
Responsive behavior (for news feed items) wasn't as nice as the new Featured Data cards, so I updated it.